### ISSUE TITLE
Report extra spaces in struct tags

### ISSUE THREAD

**Issue by jbuberel** [golang/lint#154]
Feature request to detect/report the presence of extra space in struct tags. These problems are detected using `go vet`, but seem to fall into the category of things that `golint` could detect.

http://play.golang.org/p/G2KUJj58lb

Although this program will compile and run, these struct tags will not result in the values being set on the variables as expected. Hard to detect by visual inspection.

```
type random struct {
	Type string `json: "type"`
	Size int    `json: "size"`
	Min  int    `json: "min"`
	Max  int    `json: "max"`
}
```

**Comment by dsymonds** [comment]
It's a correctness thing, not a style thing, so it is govet's domain, not golint's domain. As you said, govet flags it already; there's no point in golint also checking for it.
